docs/ebpf/concepts/rlimit.md

```diff
@@ -0,0 +1,88 @@
+# Resource Limits
+
+Creating eBPF objects (Maps, Programs, even BTF blobs) requires kernel memory
+allocation. Before kernel version 5.11, you could control the memory available
```

Comment:
Nit: i would probably phrase it as "the memory available was restricted by RLIMIT_MEMLOCK" rather than "you can control it". That helps the reader understand that RLIMIT_MEMLOCK is bad, and that it's good that it's gone.

Reply:
Good idea, done!